### PR TITLE
Updated Deltacomb 3D machine and quality definitions

### PR DESCRIPTION
**Pull request by kaleidoscopeit**

### CODE REVIEW

File: resources/definitions/deltacomb.def.json [hunk 1]
```diff
@@ -52,6 +52,9 @@
         "top_bottom_thickness":                  { "default_value": 0.6 },
         "support_z_distance":                    { "value": "layer_height * 2" },
         "support_bottom_distance":               { "value": "layer_height" },
-        "support_use_towers" :                   { "default_value": false }
+        "support_use_towers" :                   { "default_value": false },
+        "jerk_wall_0" :                          { "value": "30" },
+				"jerk_travel" :                          { "default_value": 20 },
```
Comment (diegopradogesto) [inline resources/definitions/deltacomb.def.json]: Fix indentations.

File: resources/quality/deltacomb/deltacomb_abs_High_Quality.inst.cfg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[general]
+version = 4
+name = Extra Fine (beta)
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_abs_High_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_abs_Draft_Quality.inst.cfg [hunk 3]
```diff
@@ -1,13 +1,13 @@
 [general]
 version = 4
+name = Fast (beta)
 definition = deltacomb
-name = Fast Quality (beta)
 
 [metadata]
-setting_version = 5
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_abs_Draft_Quality.inst.cfg]: master branch is currently in `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_abs_Fast_Quality.inst.cfg [hunk 4]
```diff
@@ -1,13 +1,13 @@
 [general]
 version = 4
+name = Normal (beta)
 definition = deltacomb
-name = Normal Quality (beta)
 
 [metadata]
-setting_version = 5
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_abs_Fast_Quality.inst.cfg]: master branch is currently in `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_abs_Normal_Quality.inst.cfg [hunk 5]
```diff
@@ -1,13 +1,13 @@
 [general]
 version = 4
+name = Fine (beta)
 definition = deltacomb
-name = High Quality (beta)
 
 [metadata]
-setting_version = 5
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_abs_Normal_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_abs_Superdraft_Quality.inst.cfg [hunk 6]
```diff
@@ -0,0 +1,25 @@
+[general]
+version = 4
+name = Sprint (beta)
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_abs_Superdraft_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_abs_Verydraft_Quality.inst.cfg [hunk 7]
```diff
@@ -0,0 +1,25 @@
+[general]
+version = 4
+name = Extra Fast (beta)
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_abs_Verydraft_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_global_Draft_Quality.inst.cfg [hunk 8]
```diff
@@ -0,0 +1,14 @@
+[general]
+version = 4
+name = Fast
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_global_Draft_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_global_Fast_Quality.inst.cfg [hunk 9]
```diff
@@ -0,0 +1,14 @@
+[general]
+version = 4
+name = Normal
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_global_Fast_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_global_High_Quality.inst.cfg [hunk 10]
```diff
@@ -0,0 +1,14 @@
+[general]
+version = 4
+name = Extra Fine
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_global_High_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_global_High_Quality.inst.cfg [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+type = quality
+quality_type = high
+weight = 0
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_global_High_Quality.inst.cfg]: Same `weight` as in the Normal Quality. I suggest to set `weight = 1` here.

File: resources/quality/deltacomb/deltacomb_global_Normal_Quality.inst.cfg [hunk 12]
```diff
@@ -0,0 +1,14 @@
+[general]
+version = 4
+name = Fine
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_global_Normal_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_global_Verydraft_Quality.inst.cfg [hunk 13]
```diff
@@ -0,0 +1,14 @@
+[general]
+version = 4
+name = Extra Fast
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_global_Verydraft_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_global_Superdraft_Quality.inst.cfg [hunk 14]
```diff
@@ -0,0 +1,14 @@
+[general]
+version = 4
+name = Sprint
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_global_Superdraft_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_pla_Draft_Quality.inst.cfg [hunk 15]
```diff
@@ -1,23 +1,24 @@
 [general]
 version = 4
+name = Fast
 definition = deltacomb
-name = Fast Quality
 
 [metadata]
-setting_version = 5
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_pla_Draft_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_pla_High_Quality.inst.cfg [hunk 16]
```diff
@@ -0,0 +1,24 @@
+[general]
+version = 4
+name = Extra Fine
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_pla_High_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_pla_High_Quality.inst.cfg [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+type = quality
+quality_type = high
+weight = 0
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_pla_High_Quality.inst.cfg]: Same `weight` as in the Normal Quality. I suggest to set `weight = 1` here.

File: resources/quality/deltacomb/deltacomb_pla_Normal_Quality.inst.cfg [hunk 18]
```diff
@@ -1,21 +1,21 @@
 [general]
 version = 4
+name = Fine
 definition = deltacomb
-name = High Quality
 
 [metadata]
-setting_version = 5
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_pla_Normal_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_pla_Verydraft_Quality.inst.cfg [hunk 19]
```diff
@@ -0,0 +1,24 @@
+[general]
+version = 4
+name = Extra Fast
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_pla_Verydraft_Quality.inst.cfg]: Set `setting_version = 5`

File: resources/quality/deltacomb/deltacomb_pla_Superdraft_Quality.inst.cfg [hunk 20]
```diff
@@ -0,0 +1,24 @@
+[general]
+version = 4
+name = Sprint
+definition = deltacomb
+
+[metadata]
+setting_version = 4
```
Comment (diegopradogesto) [inline resources/quality/deltacomb/deltacomb_pla_Superdraft_Quality.inst.cfg]: Set `setting_version = 5`

**Comment by kaleidoscopeit** [comment]
Thank you Diego! :)

**Pull request merged**